sar/README.md

```diff
@@ -178,8 +181,8 @@ None of the fields is required.
 | Field Name             | Data Type | XML Tag                | Description                                                  |
 | ---------------------- | --------- | ---------------------- | ------------------------------------------------------------ |
 | lut_applied            | string    | `lutApplied`           |                                                              |
-| range_look_bandwidth   | \[number] | `RangeLookBandwidth`   | Range Look Bandwidth per swath, convert to GHz if required.  |
-| azimuth_look_bandwidth | \[number] | `AzimuthLookBandwidth` | Azimuth Look Bandwidth per swath, convert to GHz if required. |
+| range_look_bandwidth   | \[number] | `RangeLookBandwidth`   | Range Look Bandwidth per swath, convert to GHz if required.                |
```

Comment:
```suggestion
| range_look_bandwidth   | \[number] | `RangeLookBandwidth`   | Range Look Bandwidth per swath in Gigahertz (GHz). |
```

Reply:
You've removed the "convert or ... if required" in this PR (which is good) so it would be consistent if we would also just write "in Gigahertz (GHz)" here :-)